resources/lang/id/forum.php

```diff
@@ -0,0 +1,268 @@
+    'slogan' => 'Bermain sendiri itu berbahaya.',
+    'subforums' => 'Subforum',
+    'title' => 'osu!community',
+
+    'covers' => [
+        'create' => [
+            '_' => 'Pasang gambar sampul',
+            'button' => 'Unggah gambar',
+            'info' => 'Ukuran gambar harus :dimensions. Anda juga dapat meletakkan gambar disini untuk mengunggah.',
+        ],
+
+        'destroy' => [
+            '_' => 'Hapus gambar sampul',
+            'confirm' => 'Apakah Anda yakin akan menghapus gambar sampul ini?',
+        ],
+    ],
+
+    'email' => [
+        'new_reply' => '[osu!] Balasan baru dari topik ":title"',
+    ],
+
+    'forums' => [
+        'topics' => [
+            'empty' => 'Topik tidak ditemukan!',
+        ],
+    ],
+
+    'post' => [
+        'confirm_destroy' => 'Yakin menghapus post?',
+        'confirm_restore' => 'Yakin mengembalikan post?',
+        'edited' => 'Terakhir disunting oleh :user :when, total disunting sebanyak :count kali.',
+        'posted_at' => 'Diposting pada :when',
+
+        'actions' => [
+            'destroy' => 'Hapu post',
+            'restore' => 'Kembalikan post',
+            'edit' => 'Sunting post',
+        ],
+    ],
+
+    'search' => [
+        'go_to_post' => 'Pergi ke post',
+        'post_number_input' => 'masukkan nomor post',
+        'total_posts' => ':posts_count post total',
+    ],
+
+    'topic' => [
+        'deleted' => 'topik dihapus',
+        'go_to_latest' => 'lihat posting terbaru',
+        'latest_post' => ':when oleh :user',
+        'latest_reply_by' => 'balasan terbaru oleh :user',
+        'new_topic' => 'Post topik baru',
+        'post_reply' => 'Post',
+        'reply_box_placeholder' => 'Ketik di sini untuk membalas',
+        'started_by' => 'oleh :user',
+
+        'create' => [
+            'preview' => 'Pratinjau',
+            // TL note: this is used in the topic reply preview, when
+            // the user goes back from previewing to editing the reply
+            'preview_hide' => 'Ketik',
+            'submit' => 'Post',
+
+            'placeholder' => [
+                'body' => 'Ketik konten post di sini',
+                'title' => 'Klik di sini untuk mengatur judul',
+            ],
+        ],
+
+        'jump' => [
+            'enter' => 'klik untuk memasukkan nomor tertentu',
+            'first' => 'pergi ke kiriman pertama',
+            'last' => 'pergi ke kiriman terakhir',
+            'next' => 'lewati 10 pos berikutnya',
+            'previous' => 'kembali 10 posting',
+        ],
+
+        'post_edit' => [
+            'cancel' => 'Batal',
+            'post' => 'Simpan',
+        ],
+    ],
+
+    'topic_watches' => [
+        'index' => [
+            'title' => 'Langganan Forum',
+            'title_compact' => 'langganan forum',
+            'title_main' => '<strong>Langganan</strong> Forum',
+
+            'box' => [
+                'total' => 'Topik yang berlangganan',
```

Comment:
"Subscribed" here should be translated into its passive equivalent in B. Indonesia as well I believe :

+                'total' => 'Topik yang dilanggan',